adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -103,39 +108,70 @@ - (BOOL)isSDKInitialized {
   return [[VungleSDK sharedSDK] isInitialized];
 }
 
-- (BOOL)isAdCachedForPlacementID:(nonnull NSString *)placementID {
-  return [[VungleSDK sharedSDK] isAdCachedForPlacementID:placementID];
+- (BOOL)isAdCachedForPlacementID:(nonnull NSString *)placementID withDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
+  GADMAdapterVungleAdType adType = [delegate adapterAdType];
+  if (adType != GADMAdapterVungleAdTypeBanner && adType != GADMAdapterVungleAdTypeShortBanner && adType != GADMAdapterVungleAdTypeLeaderboardBanner) {
```

Comment:
@joshuajplagonera Yes, MREC is not included here, because  MREC only has one fixed size.  If the delegate is MREC, the `isBannerAd` method returns YES, because MREC could be seen as a kind of Banner to some extent.  Thanks. ^_^

Reply:
@joshuajplagonera I'm really sorry, maybe my last comment is not clear. If we use `[delegate isBannerAd]` here,  this could lead to MREC call `- (BOOL)isAdCachedForPlacementID:withSize:` method, which MREC should call `- (BOOL)isAdCachedForPlacementID:` method. Because if the delegate is MREC, the `[delegate isBannerAd]` returns YES, since MREC could be seen as a kind of Banner to some extent. Thanks a lot. 😊